Can't build Unreal Client/Server on Linux

When attempting to build the Unreal project on linux / build on cloud build:

```
/home/mark/workspace/global-multiplayer-demo/game/Source/Droidshooter/DroidshooterGameMode.cpp:35:11: error: use of undeclared identifier 'FWindowsPlatformMisc'; did you mean 'FLinuxPlatformMisc'?
        ApiKey = FWindowsPlatformMisc::GetEnvironmentVariable(*FString("DS_FE_API_KEY"));
                 ^~~~~~~~~~~~~~~~~~~~
                 FLinuxPlatformMisc
/opt/unreal-engine/Engine/Source/Runtime/Core/Public/Linux/LinuxPlatformMisc.h:16:17: note: 'FLinuxPlatformMisc' declared here
struct CORE_API FLinuxPlatformMisc : public FUnixPlatformMisc
                ^
1 error generated.
```

For Googlers: [Link to build](https://console.cloud.google.com/cloud-build/builds/acc5b594-fe51-4a36-9384-103048c81d51;step=1?project=mark-world-scale)

Looks like this a platform specific API being used, that isn't cross platform.

--- a/game/Source/Droidshooter/DroidshooterGameMode.cpp
+++ b/game/Source/Droidshooter/DroidshooterGameMode.cpp
@@ -32,7 +32,7 @@ ADroidshooterGameMode::ADroidshooterGameMode()
 	}*/
 
 	AgonesSDK = CreateDefaultSubobject<UAgonesComponent>(TEXT("AgonesSDK"));
-	ApiKey = FWindowsPlatformMisc::GetEnvironmentVariable(*FString("DS_FE_API_KEY"));
+	ApiKey = FGenericPlatformMisc::GetEnvironmentVariable(*FString("DS_FE_API_KEY"));
 }
 
 void ADroidshooterGameMode::InitGame(const FString& MapName, const FString& Options, FString& ErrorMessage)

